File: CnmPro-main/react/src/schema/organization/organization.js
import * as Yup from 'yup';

const organizationSchema = Yup.object().shape({
    name: Yup.string().min(2).max(200).required('Name is Required'),
    organization: Yup.string().min(2).max(200).required('Organization Type is Required'),
    headline: Yup.string().min(2).max(200).required('Headline is Required'),
    description: Yup.string(2).min(3).required('Description is Required'),
    logo: Yup.string().min(2).max(255).required('Logo is Required'),
    location: Yup.string().max(200),
    city: Yup.string().max(20),
    number: Yup.string().min(3).max(50).required('Phone Number is Required'),
    url: Yup.string().min(3).max(255).required('Site URL is Required'),
});

export default organizationSchema;

File: CnmPro-main/dotnet/Sabio.Models/Domain/Organizations/Organization.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sabio.Models.Domain.Organizations
{
    public class Organization
    {
        public int Id { get; set; }
        public LookUp OrganizationType { get; set; }
        public string Name { get; set; }
        public string Headline { get; set; }
        public string Description { get; set; }
        public string Logo { get; set; }
        public Location Location { get; set; } 
        public string Phone { get; set; }
        public string SiteUrl { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }
        public int UserId { get; set; }
    }
}
